frontend/src/components/blocks/SearchResult.tsx

```diff
@@ -14,48 +14,67 @@ const SearchResult = ({ document }: SearchResultProps) => {
   };
   const [year, day, month] = convertDate(formatDate());
 
+  const showMatches = () => {
+    if (
+      document.document_passage_matches.length ||
+      document.document_title_match ||
+      document.document_description_match
+    ) {
+      return (
+        <>
+          <div className="w-full md:w-auto flex flex-wrap md:flex-nowrap mt-2 md:mt-0 ">
+            {/* TODO: translate below text, how to handle plurals? */}
```

Comment:
Can be achieved through `i18n` plurals: https://www.i18next.com/translation-function/plurals

Reply:
Good to know. Might be good to make a note to create a separate ticket/task for this.